ne/README.md

```diff
@@ -2,17 +2,17 @@
+कम्प्युटर, आज को समयमा सबैले हरेक दिन सुनिरहने र प्रयोग गरी रहने साधन, बिसेगरी त्यसको छिटो र सही रुपमा काम गर्न सक्ने क्षमताको करण। कति प्राय मनिसहरुको लागि त झन कम्प्युटर जीवन जिउने माध्यम नै बनिसकेको छ। यस्को बास्तविक परिचय त त्यति बेला हुन्छ जब हामी यसलाई हामीले सोच्न सक्ने सम्म हरेक ठाउँमा  पाउछाैँ। चाहे त्यो अस्पतालको गम्भीर कामको लागी होस या त घरमा मनोरञ्जनात्मक कार्य को लागी होस, यस्को जातिल कार्य र ठूला डाटा प्रक्रिया सेकेन्डमा नै गर्न सकने क्षमताले आज कम्प्यूटर हरेक गोजीहरुमा हटाएका छ। तर यो कम्प्युटर आफ्नैमा कसरी काम गर्छ? के तपाई यो सोच्दै हुनुहुन्छ? यो बुझ्नाको लागी कम्प्यूटरलाई हामी हाम्रो दिमागसँग तुलना गरी हर्ओँ हैता।
 
 Programming involves creating a set of instructions, called a program, for a computer to follow. It can be tedious and frustrating at times because computers are very precise and need specific instructions in order to complete tasks.
 
-प्रोग्रामिंगले निर्देशनहरूको सेट सिर्जना गर्दछ । जसलाई प्रोग्राम भनिन्छ र यो कम्प्युटरको अनुसरण गर्नको लागि लेखिन्छ । यो लेखन कार्य कहिलेकाहीं थकित र निराशाजनक हुन सक्छ किनभने कम्प्युटरहरू धेरै सटीक छन् र कार्यहरू पूरा गर्न विशिष्ट निर्देशनहरू आवश्यक पर्दछ।
+कम्प्यूटर भनेको हाम्रो खोपडी जस्तै नै हो, र यस्मा प्रयोग हुने विशेष चिप, “Central Chip” भनेको हाम्रो दिमाग जस्तै। यस चिप भित्र बिभिन्ना कुराहरु सिकाइको हुन्छ। झ मजाले वन्नु पर्दा कुनै पनि कामलाई step wise step हल गर्न ट्रेन गरियेको हुन्छ, हाम्रो दिमग जास्तै । हामीलाई पनि त सानो छदा खाना खान सिकाइन्थियो, लेख्न सिकाइन्थियो र अलि अलि सिकाइयेको भरमा नै हामी आफैँ सिक्ना खोज्हेम र आझै त्यो कार्य गर्न सिकदै जानथेम। उधारणको लागी खाना पकाउने बेला हामीलाई एउटा निश्चित निर्देशन दिएको हुन्छ कि पैला कुकर लिने, नी त्यसमा पुग्ने गरि चामल हल्ने र त्यस्मा अडकलेर पानी थाप्ने, त्यस पछि ग्यास बालेर कुकर बसौने र पाकी नसकेसम्म सीटी बजाउन दिनुहोस्। तर यसलाइ गर्ने अर्को नी तारिका हुन सक्छ, चमल र पानी ठिक गरेर इलेक्ट्रिक राइस कुकरमा हल्ने र पावर अन गर्दिन। हो यसतै केही काम गर्ने set of instructions लाइ नै हामी कम्प्यूटरको भासामा प्रोग्राम(Program)। यो मध्धे कुनै पनि स्टेप बिग्रियो भने हामीले सोचेको जस्तै मिठो खाना नपक्न सक्छ। त्यसै गरि program लेख्दा नी कुनै पनि ठाँउमा गल्ती भयो भने हामीले सोचेको जस्तो output न आउना सक्छ । र खानालाइ पनि प्रेसर कुकर र राइस कुकरमा पकाए त्यस्लाई अटोमेटिक या स्वाडिलो बनाउन सकिने जस्तै कुनै पनि कम्प्यूटर प्रोग्राम लाई बिभिन्न तारिकाले लेखेर त्यस्को चल्ने गतिलाई बधौने र त्यो प्रोग्राम चलौने मान्छेलाई अझ सजिलो बनाउने गर्न सकिन्छ। हो खाना पकौदा लग्ने सबाइ अनुभव कम्प्यूटर प्रोग्राम लेखनमा नि लाग्छ। सानो सानो काम गरना अलक्सी लगने, पकौदै गर्दा स्वाड बिग्रिए रिस उठ्ने र स्वाद मिलौन अर्को झनझट हुन तर साही ढंगाले गरे पाकी सक्दा स्वाड  को आनन्द लिन छुटै नै मजा भए जस्तै program लेख्दा नी errorहरु फेला पार्न एकदम कथीन र ठकौ लाग्ने काम हुन सक्छ तर जब त्यो program एक छोटी पुरा भएर चल्न थाल्छ, त्यसको आनन्द नै छुटै हुन्छ।
```

Comment:
I think we have couple typos here :
`बिभिन्ना `: `विभिन`, `यस्मा `: `यसमा`, `उधारणको`: `उदाहरणका`, `गर्दिन`: `गर्दिने `, `errorहरु`: `त्रुटिहरू`

Can you help me to understand this sentence?
`झ मजाले वन्नु पर्दा कुनै पनि कामलाई step wise step हल गर्न ट्रेन गरियेको हुन्छ`

Reply:
I have intentionally kept the word "error" because it is more easy and kind of unchangeable. I think that some words should remain in English than in Nepali.

I will fix the other typos rn.

The line is trying to say that the chips are trained to solve any problem step wise step.